src/main/kotlin/org/rust/lang/core/macros/tt/FlatTree.kt

```diff
@@ -111,9 +111,12 @@ private class FlatTreeBuilder {
     }
 
     private fun subtree(subtreeId: Int, subtree: TokenTree.Subtree) {
-        var firstTt = tokenTree.size()
+        var firstTt = tokenTree.size
         val nTt = subtree.tokenTrees.size
-        tokenTree.fill(firstTt, firstTt + nTt, -1)
+        tokenTree.ensureCapacity(firstTt + nTt)
+        for (i in firstTt until firstTt + nTt) {
+            tokenTree.set(i, -1)
+        }
```

Comment:
Maybe
```kotlin
tokenTree.elements().fill(firstTt, firstTt + nTt, -1)
```

Reply:
Unfortunately, this doesn't work, just like my previous code did not work as well :(